generate.coffee

```diff
@@ -86,8 +87,8 @@ generateAddresses = (total) ->
     address =
       entity_id: index
       city: chance.city()
-      state: chance.state()
-      country: chance.country({full:true})
+      region: regions.region_id[chance.state()]
```

Comment:
I don't think you need a region id here. This field is a varchar and from querying a magento client's DB i get this kind of data:

```
entity_id	country_id	region
1	US	Alabama
2	US	Alabama
23	US	New York
24	US	New York
25	US	New York
26	US	New York
27	US	New York
28	US	New York
29	US	New York
30	US	New York
31	US	North Carolina
32	US	North Carolina
33	US	Oregon
34	US	Oregon
35	US	Oregon
36	US	Oregon
```